castai/resource_node_template.go

```diff
@@ -368,10 +382,37 @@ func resourceNodeTemplate() *schema.Resource {
 										Description: "Availability zone name.",
 									},
 									FieldNodeTemplateName: {
-										Required:    true,
+										Optional:    true,
```

Comment:
Intended change?

Reply:
Yes, you don't need to provide the dedicated node group name if you are using the selectors